dp/cloud/go/services/dp/storage/cbsd_manager_test.go

```diff
@@ -112,6 +114,8 @@ func (s *CbsdManagerTestSuite) TestCreateCbsd() {
 		s.Require().NoError(err)
 
 		cbsd := getBaseCbsd()
+		cbsd.CbsdCategory = db.MakeString("b")
```

Comment:
Are there any test for other methods (Update/Fetch)? Also it look like were are just testing default values